release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -655,6 +655,18 @@ Operator authors can now use the Ansible-based Operator support in the Operator
 
 For more details, see xref:../operators/operator_sdk/osdk-monitoring-prometheus.adoc#osdk-monitoring-prometheus[Exposing custom metrics for Ansible-based Operators].
 
+==== Object pruning for Go-based Operators
+
+The `operator-lib` pruning utility lets Go-based Operators clean up objects, such as jobs or pods, that can stay in the cluster and use resources. The utility includes common pruning strategies for Go-based Operators. Operator authors can also use the utility to create custom hooks and strategies.
```

Comment:
Do Kubernetes resources like Jobs or Pods need to be capitalized?

Reply:
Sounds like the rule is do not capitalize pods which likely extends to jobs as well. 
/lgtm